--- mariadb-10.1.20/scripts/mysqld_safe.sh.original	2017-01-10 14:15:26.741942544 -0500
+++ mariadb-10.1.20/scripts/mysqld_safe.sh	2017-01-10 14:16:07.389782751 -0500
@@ -768,7 +768,7 @@
 then
   if ! `mkdir -p $mysql_unix_port_dir`
   then
-    log_error "Fatal error Can't create database directory '$mysql_unix_port'"
+    echo "Fatal error Can't create database directory '$mysql_unix_port'"
     exit 1
   fi
   chown $user $mysql_unix_port_dir
@@ -987,10 +987,6 @@
 # Avoid 'nohup: ignoring input' warning
 test -n "$NOHUP_NICENESS" && cmd="$cmd < /dev/null"
 
-# close stdout and stderr, everything goes to $logging now
-exec 1>&-
-exec 2>&-
-
 log_notice "Starting $MYSQLD daemon with databases from $DATADIR"
 
 # variable to track the current number of "fast" (a.k.a. subsecond) restarts
--- mariadb-10.1.20/support-files/mysql.server.sh.original	2017-01-10 14:17:17.986505225 -0500
+++ mariadb-10.1.20/support-files/mysql.server.sh	2017-01-10 14:17:56.641353267 -0500
@@ -308,7 +308,7 @@
     then
       # Give extra arguments to mysqld with the my.cnf file. This script
       # may be overwritten at next upgrade.
-      $bindir/mysqld_safe --datadir="$datadir" --pid-file="$mysqld_pid_file_path" "$@" &
+      $bindir/mysqld_safe --datadir="$datadir" --pid-file="$mysqld_pid_file_path" "$@" >dev/null &
       wait_for_ready; return_value=$?
 
       # Make lock for RedHat / SuSE
